packages/nimble-components/src/anchor-tabs/specs/README.md

```diff
@@ -0,0 +1,206 @@
+
+None
+
+### Prior Art/Examples
+
+In SLE:
+
+![SLE](sle-systems.png)
+
+In GitHub:
+
+![GitHub](github.png)
+
+---
+
+## Design
+
+Anchor tabs have the same appearance as a standard tabs control, i.e. a sequence of operable regions with labels, each of which corresponds to some target content to display. In the case of anchor tabs, they do not switch between pre-configured tab panels, but rather navigate to some URL. The tabs can be configured to load the URL in a specific target, like an `iframe`.
+
+The design consists of an "Anchor tabs" container component which contains one or more "Anchor tab" components. This mirrors the existing design of the `nimble-tabs` and `nimble-tab` components, however there is no analog for the `nimble-tab-panel` component. Instead, the content loaded from the configured URL(s) loads into the containing page, an `iframe`, a new window, etc.
+
+We cannot extend FAST's tabs control because it has baked into it the idea that tabs activate tab panels. Instead we will create new components from scratch, but we can likely borrow logic from FAST e.g. for handling keyboard navigation.
+
+### API
+
+#### Anchor Tabs
+
+- *Component Name* - `nimble-anchor-tabs`
+- *Props/Attrs*
+    -   `activeid` - id of active tab. Defaults to the first tab.
+- *Methods* - None
+- *Events*
+    - `change` - fired when the active tab changes
+- *CSS Classes and CSS Custom Properties that affect the component* - None
+- *Slots*
+    -   `start` - content placed to the left of tab list
+    -   `anchortab` - slot for anchor tab elements
+    -   `end` - content placed to the right of tab list
+
+#### Anchor Tab
+
+- *Component Name* - `nimble-anchor-tab`
+- *Props/Attrs*
+    -   `download` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes)
+    -   `href` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes).
+    -   `hreflang` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes)
+    -   `ping` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes)
+    -   `referrerpolicy` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes)
+    -   `rel` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes)
+    -   `target` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes)
+    -   `type` - see [HTML anchor doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes)
+    -   `disabled` - disables the tab (this matches the API of the existing `nimble-tab`)
+- *Methods* - None
+- *Events* - None
+- *CSS Classes and CSS Custom Properties that affect the component* - None
+- *Slots*
+    - *default*
+
+### Anatomy
+
+#### Anchor Tabs
+
+```html
+<template>
+    <slot name="start"></slot>
+    <nav class="tablist">
+        <ul>
```

Comment:
For my education, can you explain why you chose `<ul>` (unordered list) instead of `<ol>` (ordered list) or something else? Is this pattern borrowed from somewhere or was it a conscious choice for this spec?

Reading the [MDN article for `ul`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul) I think it's a better choice than `<ol>`; while a client may choose to order their tabs by importance, it doesn't change the meaning if they were in a different order.

>The `<ol>` and `<ul>` elements both represent a list of items. They differ in that, with the `<ol>` element, the order is meaningful. To determine which one to use, try changing the order of the list items; if the meaning is changed, the `<ol>` element should be used, otherwise you can use `<ul>`.

Reply:
Turns out this is moot now. I was borrowing this pattern from GitHub's implementation of navigation tabs, but we've decided that we should use the `tablist` and `tab` accessibility roles instead of `navigation` and `link`, so we won't be following GitHub's example anymore.